src/App.jsx

```diff
@@ -1,13 +1,18 @@
-import "./App.css"
-import Header from "./components/client/common/header/Header.jsx"
-import HeroSection from "./components/client/landing/heroSection/HeroSection.jsx"
+import './App.css'
+import Header from './components/client/common/header/Header.jsx'
```

Comment:
Better if we can use index file instead of Header.jsx
then import file as i
import Header from './components/client/common/header'
Please follow this as a best practise for below imports also

Reply:
I didn't do this part. <HeroSection /> and <Header /> was already there when I cloned the repo. Should I change it?

Reply:
Yes Please change